src/blocks/HeaderBlockBrandPivot/Banner.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {
+  ColorComponent,
+  MarkdownHtmlComponent,
+} from 'src/blocks/BaseBlockProps'
+import {
+  CONTENT_MAX_WIDTH_DEPRECATED,
+  SectionWrapper,
+  TABLET_BP_UP,
+} from '../../components/blockHelpers'
+
+const BannerContent = styled('div')({
```

Comment:
So as with the object vs template string syntax that Gustav commented on below it's also mixed between `styled('div')` syntax and `styled.div` but we try to replace the former with the latter :)
```suggestion
const BannerContent = styled.div({
```